cfme/tests/cloud/test_tenant_quota.py

```diff
@@ -237,3 +237,83 @@ def test_service_cloud_tenant_quota_with_default_entry_point(request, appliance,
     provision_request.wait_for_request(method='ui')
     request.addfinalizer(provision_request.remove_request)
     assert provision_request.row.reason.text == "Quota Exceeded"
+
+
+def create_instance(appliance, provider, template_name):
+    instance = appliance.collections.cloud_instances.instantiate(random_vm_name('pwr-c'),
+                                                                 provider,
+                                                                 template_name)
+    if not instance.exists_on_provider:
+        instance.create_on_provider(allow_skip="default", find_in_cfme=True)
+    return instance
+
+
+@pytest.fixture(scope="function")
+def testing_instance(appliance, provider, small_template, setup_provider):
+    """ Fixture to provision instance on the provider
+    """
+    instance = create_instance(appliance, provider, small_template.name)
+    yield instance
+    instance.cleanup_on_provider()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1473325, 1506471])
```

Comment:
BZ 1506471 is closed as wontfix but it is implemented , correct ? The instance reconfigure does go through request page .
So this test case is automating that BZ.

So @pytest.mark.meta(automates=[BZ(1506471)])

and Since BZ 1473325 is not implemented yet it can just be in the Bugzilla docblock .

Reply:
- This instance reconfiguration does not go through **service/request** page. Because BZ 1506471 is not fixed.
- Also BZ 1473325 is not yet fixed but is on NEW state. 
- So I am not automating these BZs scenarios in this test case. ** This test case will get updated in the future, if these BZs got fixed. Hence added BZs into coverage.**
- This test case includes current scenario: **Quota is not honoured by instance(cloud providers) reconfiguration**

Reply:
@ganeshhubale In this case, since the test case is not yet providing test coverage, I recommend leaving them in the `Bugzilla` docblock with a TODO to add coverage for them, which you've already done. I wouldn't put them in the `coverage` list though.

Reply:
Agree! Since those BZs are not yet fixed. But also existing scenario is valid. So I have added this test case. I updating with adding these BZs in doc block. I already had added TODO to updated this test case once BZs fixed.

Reply:
>     * This instance reconfiguration does not go through **service/request** page. Because BZ 1506471 is not fixed.
> 
>     * Also BZ 1473325 is not yet fixed but is on NEW state.
> 
>     * So I am not automating these BZs scenarios in this test case.
> 
>     * **This test case will get updated in the future, if these BZs got fixed. Hence added BZs into coverage.**
> 
>     * This test case includes current scenario: **Quota is not honoured by instance(cloud providers) reconfiguration**

@ganeshhubale : But you are checking status of request in your test 311 to 313 . Reconfigure instance gets a request right ?